### ISSUE TITLE
run.spec seems to be unix-specific in version 2019.1

### ISSUE THREAD

**Issue by ArndalAndersen** [psyfood/pyqmix-web#38]
Quick-fix is to run the `pyinstaller --onefile run.py` command as stated in our [documentation](https://github.com/psyfood/pyqmix-web/blob/master/EXECUTABLE.md) which will rebuild the `run.spec` in Windows format.

**Comment by hoechenberger** [comment]
What does that mean, unix-specific? The line endings or what?

**Comment by ArndalAndersen** [comment]
The issue was that I could clone this `pyqmix-web` repository and run the command `pyinstaller --clean run.spec` to successfully create a standalone using my mac. When I used my Windows computer, however, I would get an error doing the exact same thing. On Windows I had to recreate the `run.spec` file and only _then_ would the `pyinstaller --clean run.spec`-command successfully create an executable. 

I created the `run.spec`-file on my mac, so I just figured that's why Windows couldn't use the `run.spec`-file? Anyhow, `pyqmix-web` is primarily targeted for Windows computers at the moment (due to pyqmix) so I thought I'd create an issue and say that the `run.spec` file should be generated by a Windows computer instead of the current one that is created by a Mac. 

tl;dr: I'm messing things up developing both on a mac and a Windows computer

**Comment by hoechenberger** [comment]
Then I suggest we compare both files and see how they are different. An easy way to do that is by putting them both in the same folder (naming one file `run.spec.macos` and the other `run.spec.win` or so), opening that folder in PyCharm, selecting both files (click on the first and then Cmd-Click on the second), and select `Compare Files` from the context menu)

**Comment by ArndalAndersen** [comment]
Cool. Just did that, and the `run.spec` files are identical besides, of course, the `pathex` file path. I might just have specified it wrong when I tested it. 
I just successfully created a standalone with the run.spec (v2019.1) on my mac, so I'll close this issue now.

**Comment by hoechenberger** [comment]
> files are identical besides, of course, the `pathex` file path.
Oh, how do they differ? I don't see any reason why they _should_ differ?

> I just successfully created a standalone with the run.spec (v2019.1) on my mac, so I'll close this issue now.

So does this mean you can use the same `.spec` file to build on _both_, macOS _and_ Windows?

**Comment by ArndalAndersen** [comment]
> Could you now use the exact same run.spec to build on both, macOS and Windows? 

Yes, exactly.

**Comment by hoechenberger** [comment]
Okay, Great!
